contracts/contracts/ValidatorSet.sol

```diff
@@ -67,6 +69,14 @@ contract ValidatorSet {
         return true;
     }
 
+    function getLengthOfEpochStartHeights() external view returns(uint) {
```

Comment:
Is this the number of epochs? If so, call it `numberOfEpcohs` or `getNumberOfEpochs`.

Reply:
Yeah, this ugly name was for sticking to the name of the variable. But related to trustlines-protocol/blockchain#74 we probably provide much more comfortable functions and make the variables not public.

Reply:
So I go for `getEpochStartHeights()` and `getValidators(uint _epochStart)`. Your thoughts?

Reply:
`getEpochStartHeights()` is incorrect, you don't get the height of an epoch start, you get the number of epochs. So `getNumberOfEpochs` seems to be the optimal name for me.

Reply:
No, I changes it to 
```solidity
function getEpochStartHeights() external view returns(uint[]) {
    return epochStartHeights;
}
```
 Still think it is incorrect?

Reply:
Checkout the last commit. Else I overwrite it.

Reply:
Sure, if you change what the function does, now it is fine :).